python/taichi/lang/__init__.py

```diff
@@ -396,7 +396,7 @@ def __init__(self):
         self.gdb_trigger = False
         self.excepthook = False
         self.experimental_real_function = False
-        self.experimental_ast_refactor = False
+        self.experimental_ast_refactor = True
```

Comment:
IIUC, after this PR, `experimental_ast_refactor` will turn on by default?

Reply:
IIUC it can be turned on by default if no more bugs are found? @k-ye @ailzhang

Reply:
MeshTaichi is now rely on the old AST generator, better if turn it on when MeshTaichi merges back and migrates to the new AST generator?

Reply:
@lin-hitonami Let's keep experimental_ast_refactor=False in this PR so that @g1n0st can give it a try to rebase MeshTaichi. 
@g1n0st Would you mind giving it a try after PR merges and work with @lin-hitonami to migrate MeshTaichi to this new ast generator asap? We need to get rid of the old ast generator soon otherwise we'll have to maintain both paths ;)

Reply:
Sure~ I will try it next week and work with @lin-hitonami . : )